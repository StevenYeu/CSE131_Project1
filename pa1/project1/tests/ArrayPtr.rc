function : void main() {
    int x;
    x[99] = x[-40] + x[*x];
}


