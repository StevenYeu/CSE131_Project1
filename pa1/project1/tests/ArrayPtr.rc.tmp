Error, "project1/tests/ArrayPtr.rc": 
  Type of expression referenced by array subscript (int) is not of array or pointer type.
Error, "project1/tests/ArrayPtr.rc": 
  Type of expression referenced by array subscript (int) is not of array or pointer type.
Error, "project1/tests/ArrayPtr.rc": 
  Incompatible type int to unary dereference operator *, pointer expected.
Compile: failure.
