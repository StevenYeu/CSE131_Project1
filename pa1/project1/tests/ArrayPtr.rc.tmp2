  Type of expression referenced by array subscript (int) is not of array or pointer type.
  Type of expression referenced by array subscript (int) is not of array or pointer type.
  Incompatible type int to unary dereference operator *, pointer expected.
Compile: failure.
