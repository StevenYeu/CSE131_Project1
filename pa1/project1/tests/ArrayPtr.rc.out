Error, "/tmp/cs131_testrunner_QI0qMF/file.rc": 
  Type of expression referenced by array subscript (int) is not of array or pointer type.
Error, "/tmp/cs131_testrunner_QI0qMF/file.rc": 
  Type of expression referenced by array subscript (int) is not of array or pointer type.
Error, "/tmp/cs131_testrunner_QI0qMF/file.rc": 
  Incompatible type int to unary dereference operator *, pointer expected.
Compile: failure.
